comparison_blogpost.txt

```diff
@@ -0,0 +1,59 @@
+Summary
+Overall, Apache Airflow is the heavier, more feature-dense and battle-tested service of the two, while Versatile Data Kit is more minimal and lightweight in comparison without sacrificing any fundamental capability and while offering a higher level of extensibility.
+
+Comparison points:
+
+Pricing
+Scheduling
+Simplicity/Ease of use
+Workloads
+Language
+
+
+Pricing
+
+Both are free and open-source. Additionally, both Apache Airflow and Versatile Data Kit are licensed under the Apache License Version 2.0.
+
+Workload Scheduling
+
+Versatile Data Kit schedules workloads through a cron-like interface part of a Data job’s configuration.
+Apache Airflow allows you to schedule DAG runs using a cron string, but also using a `datetime.timedelta` object. The first DAG run will trigger according to the minimum start_date value within the DAG, while subsequent runs will be triggered according to the aforementioned schedule.
+Additionally, Airflow’s DAG structure allows you to schedule workloads in an interdependent manner, meaning a certain workload will only be executed if a previous workload or set of workloads was executed successfully first.
+
+Simplicity/Ease of use
+
+Both projects offer a Helm chart which allows users to deploy the respective services to a Kubernetes cluster. However, by default Airflow uses a SQLite database as its storage, which is not recommended for a production deployment. Users must configure and manage another database service, such as an instance of PostgreSQL, and configure Airflow to connect to it.
+Airflow offers both a command-line interface, and a web UI for managing workloads. Versatile Data Kit only offers a command-line interface.
+Overall, Apache Airflow is the more feature-dense service, while Versatile Data Kit is more minimal, lightweight and overall simpler to use.
+
+Workloads
+
+Versatile Data Kit allows you to deploy and run workloads consisting of Python files, SQL scripts, or a mixture of both.
+Apache Airflow workloads can be composed of many different types of operators which serve a wider pool of functions compared to Versatile Data Kit.
```

Comment:
Airflow is more generic. 
While with VDK you can run any python/sql workload. It is very much optimized for ELT/ETL type of work with interfaces for ingestion of data, transformation.